rust-symcrypt/src/lib.rs

```diff
@@ -82,8 +83,7 @@ use std::sync::Once;
 
 pub mod block_ciphers;
 pub mod chacha;
-pub mod ecdh;
-pub mod eckey;
+pub mod ecc;
```

Comment:
How do you handle versioning for this library? Changing the name of this module would be a breaking change, right?

Reply:
We have not update the version since we first released the crate. 

What I aim to do is do a major change with all of the updated code that we've added and make note of the breaking change. 

We would change from `0.2.0` to `0.3.0`, and version `0.2.0` would still work fine.